src/StatusQ/Controls/StatusSwitchTabBar.qml

```diff
@@ -0,0 +1,16 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import StatusQ.Core.Theme 0.1
+
+TabBar {
+    id: control
+    padding: 1
+
+    background: Rectangle {
+        implicitHeight: 36
+        color: Theme.palette.directColor7
+        radius: 8
+        border.width: 1
+        border.color: color
```

Comment:
border have a color the same as whole rectangle?

Reply:
Yes. There's a padding there as well, so we'll see the border around the tabs